geomstats/information_geometry/normal.py

```diff
@@ -116,8 +160,420 @@ def pdf(x):
         return pdf
 
 
-class NormalMetric(PullbackDiffeoMetric):
-    """Class for the Fisher information metric on normal distributions.
+class CenteredNormalDistributions(InformationManifoldMixin, SPDMatrices):
+    """Class for the manifold of centered multivariate normal distributions.
+
+    This is the class for multivariate normal distributions with zero mean.
+    Each distribution is represented by its covariance matrix, i.e. a symmetric
+    positive-definite matrix of size :math:`sample_dim`.
+
+    Parameters
+    ----------
+    sample_dim : int
+        Dimension of the sample space of the multivariate normal distribution.
+    """
+
+    def __init__(self, sample_dim):
+        super().__init__(n=sample_dim)
+        self.sample_dim = sample_dim
+        self.metric = CenteredNormalMetric(sample_dim=sample_dim)
+
+    def sample(self, point, n_samples=1):
+        """Sample from a centered multivariate normal distribution.
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., sample_dim, sample_dim]
+            Symmetric positive definite matrix representing the covariance matrix
+            of a multivariate normal distribution with zero mean.
+        n_samples : int
+            Number of points to sample with each covariance matrix in point.
+            Optional, default: 1.
+
+        Returns
+        -------
+        samples : array-like, shape=[..., n_samples, sample_dim]
+            Sample from centered multivariate normal distributions.
+        """
+        geomstats.errors.check_belongs(point, self)
+        if point.ndim > 3:
+            raise NotImplementedError
```

Comment:
in other parts of the code we decided to not make any check. Should we be consistent?